docs/getting-started/rulesets.md

```diff
@@ -80,11 +80,9 @@ Specifying the format is optional, so you can completely ignore this if all the
 rules:
   api-servers:
     description: "OpenAPI `servers` must be present and non-empty array."
-    recommended: true
     formats: ["oas3"]
-    given: "$"
+    given: "$.servers"
```

Comment:
Noticed a few of these. We want to get as close to the field as possible to make it show the right line on Studio.

Reply:
`$` combined with `server` field is fully compliant with `$.servers`, so we need to be careful about changing them.
The biggest difference is that a function won't be run if property matching `$.servers` does not exist in the document, while it'd be run if `field` was specified.
cc @nulltoken since you are working on that too.
I'm pretty sure `src/rulesets/oas3/__tests__/api-servers.ts` fails now.

Reply:
That said, we could simply add `given` to the UI. It's not a big chunk of work for me, and I'd feel a bit safer. :sweat_smile:. We're all working on a huge chunk of work around OpenAPI rulesets, hence it might be trivial to sneak a small regression in, as there is plenty of code to review.

Reply:
@philsturgeon As #773 takes over this PR, would you agree to close this one and move the discussion over to #773?

>cc @nulltoken since you are working on that too.
>I'm pretty sure src/rulesets/oas3/__tests__/api-servers.ts fails now.

@P0lip I don't see it failing in #773. Or is it passing for the wrong reason? Or did I  misunderstand you?

Reply:
@nulltoken I think the plan is to merge #773 into here then the two of you can continue to make progress over here.

Reply:
@philsturgeon I'm afraid this won't work as this PR is linked to a branch in the core repository. And I'm not granted with push bit in this one.

I've already cherry picked your commits in my branch and switched the base of #773 to target `develop`.

It should be safe to close this one and move over to #773. Your work wouldn't be lost and @P0lip and I could work there.

Thoughts?

Reply:
Sounds good, good luck!